README.md

```diff
@@ -1,46 +1,62 @@
 
-In the project directory, you can run:
+## Built With
 
-### `npm start`
+- Major languages: HTML | CSS | JavaScript 
+- Library/Framework: Reactjs | Redux | Jest 
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
+## Live Demo
+[Live Demo Link](https://deploy-preview-7--vocal-praline-119922.netlify.app/)
 
-The page will reload when you make changes.\
-You may also see any lint errors in the console.
+## Project Video
+[Project Video Link](https://youtu.be/viAMZGRw9d4)
 
-### `npm test`
 
-Launches the test runner in the interactive watch mode.\
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+## Preview
+![space](./screenshot.png)
 
-### `npm run build`
+## Getting started
 
-Builds the app for production to the `build` folder.\
-It correctly bundles React in production mode and optimizes the build for the best performance.
+#### Clone this repository
 
-The build is minified and the filenames include the hashes.\
-Your app is ready to be deployed!
+```bash
+$ git clone https://github.com/AckonSamuel/watch-hub.git
+```
 
-See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
+#### cd into directory
+```
+$ cd <relative-path>/watch-hub/
+```
 
-### `npm run eject`
+#### install dependencies
+``` run 
+$ npm i
+```
+#### run project
+```
+$ npm start
+```
 
-**Note: this is a one-way operation. Once you `eject`, you can't go back!**
+## Author
 
-If you aren't satisfied with the build tool and configuration choices, you can `eject` at any time. This command will remove the single build dependency from your project.
+👤 **Ackon Samuel**
+- GitHub: [@AckonSamuel](https://github.com/AckonSamuel/)
+- Twitter: [@AckonSamuel2](https://twitter.com/AckonSamuel2)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/samuel-yaw-ackon/)
 
-Instead, it will copy all the configuration files and the transitive dependencies (webpack, Babel, ESLint, etc) right into your project so you have full control over them. All of the commands except `eject` will still work, but they will point to the copied scripts so you can tweak them. At this point you're on your own.
+👤
+## 🤝 Contributing
 
-You don't have to ever use `eject`. The curated feature set is suitable for small and middle deployments, and you shouldn't feel obligated to use this feature. However we understand that this tool wouldn't be useful if you couldn't customize it when you are ready for it.
+Contributions, issues, and feature requests are welcome!
 
-## Learn More
+Feel free to check the [issues page](../../issues/).
 
-You can learn more in the [Create React App documentation](https://facebook.github.io/create-react-app/docs/getting-started).
+## Show your support
 
-To learn React, check out the [React documentation](https://reactjs.org/).
+Give a ⭐️ if you like this project!
```

Comment:
- [ ] I suggest adding the link to your `MIT.md` file to your README 👌, and remember to add your name and customize the MIT.md  😉

![image](https://user-images.githubusercontent.com/45231276/187921179-446494ab-ceed-4de9-8627-0c8646bda84b.png)


> The MIT license gives users express permission to reuse code for any purpose, sometimes even if the code is part of proprietary software. As long as users include the original copy of the MIT license in their distribution, they can make any changes or modifications to the code to suit their own needs. 💯

Reply:
Got it!

Reply:
Thanks for the review